manifests/d/DarkfullDante/wol/1.0.2/DarkfullDante.wol.yaml

```diff
@@ -0,0 +1,28 @@
+PackageIdentifier: "DarkfullDante.wol"
+PackageVersion: "1.0.2"
+PackageLocale: "en-US"
+Publisher: "DarkfullDante"
```

Comment:
Publisher needs to match `A`dd or `R`emove `P`rograms
```suggestion
Publisher: "Felix Cusson"
```

Reply:
Hi @OfficialEsco,
I didn't have to do that change on my last addition to winget (I copied my last manifest from my other software). Should I make the change on my next update for srt-align?

Reply:
change done

Reply:
Well thats because the Publisher was correct.
![image](https://user-images.githubusercontent.com/15158490/134669180-d6041942-2d8e-441c-b4d4-d81c40cc587b.png)

Reply:
well I'll be dammed, might have not been comfortable to put my name on my first publicly released software. Thank you for the info